libs/langchain-azure-cosmosdb/src/caches.ts

```diff
@@ -0,0 +1,172 @@
+import {
+  BaseCache,
+  deserializeStoredGeneration,
+  getCacheKey,
+  serializeGeneration,
+} from "@langchain/core/caches";
+import { Generation } from "@langchain/core/outputs";
+import { Document } from "@langchain/core/documents";
+import { EmbeddingsInterface } from "@langchain/core/embeddings";
+import { CosmosClient, CosmosClientOptions } from "@azure/cosmos";
+import { DefaultAzureCredential } from "@azure/identity";
+import { getEnvironmentVariable } from "@langchain/core/utils/env";
+import {
+  AzureCosmosDBNoSQLConfig,
+  AzureCosmosDBNoSQLVectorStore,
+} from "./azure_cosmosdb_nosql.js";
+
+const USER_AGENT_SUFFIX = "LangChain-CDBNoSQL-SemanticCache-JavaScript";
+const DEFAULT_DATABASE_NAME = "CosmosNoSqlCacheDB";
```

Comment:
```suggestion
const DEFAULT_DATABASE_NAME = "semanticCacheDB";
```
Aligning on other integrations.

@aayush3011 I was thinking that it might be even better for the DB to use a default common one for the VectorSearch and SemanticCache integration? In that case, you could just remove the default DB name here and let it fall back to the vector search one if it's not defined.

This way users would by default have a unique DB, with one container for each use case, simplifying the setup.